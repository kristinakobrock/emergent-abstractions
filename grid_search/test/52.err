  0%|                                                                                                     | 0/50 [00:00<?, ?it/s]  6%|#####5                                                                                       | 3/50 [00:00<00:01, 24.20it/s] 12%|###########1                                                                                 | 6/50 [00:00<00:01, 26.02it/s] 20%|##################4                                                                         | 10/50 [00:00<00:01, 29.66it/s] 26%|#######################9                                                                    | 13/50 [00:00<00:01, 26.21it/s] 32%|#############################4                                                              | 16/50 [00:00<00:01, 25.86it/s] 40%|####################################8                                                       | 20/50 [00:00<00:01, 27.66it/s] 48%|############################################1                                               | 24/50 [00:00<00:00, 28.19it/s] 54%|#################################################6                                          | 27/50 [00:01<00:00, 26.31it/s] 60%|#######################################################2                                    | 30/50 [00:01<00:00, 27.27it/s] 66%|############################################################7                               | 33/50 [00:01<00:00, 26.77it/s] 72%|##################################################################2                         | 36/50 [00:01<00:00, 25.39it/s] 78%|#######################################################################7                    | 39/50 [00:01<00:00, 23.57it/s] 84%|#############################################################################2              | 42/50 [00:01<00:00, 24.03it/s] 92%|####################################################################################6       | 46/50 [00:01<00:00, 25.91it/s] 98%|##########################################################################################1 | 49/50 [00:01<00:00, 25.77it/s]100%|############################################################################################| 50/50 [00:01<00:00, 25.92it/s]
  0%|                                                                                                     | 0/13 [00:00<?, ?it/s] 15%|##############3                                                                              | 2/13 [00:00<00:00, 19.41it/s] 46%|##########################################9                                                  | 6/13 [00:00<00:00, 28.24it/s] 69%|################################################################3                            | 9/13 [00:00<00:00, 25.06it/s] 92%|####################################################################################9       | 12/13 [00:00<00:00, 23.81it/s]100%|############################################################################################| 13/13 [00:00<00:00, 24.56it/s]
