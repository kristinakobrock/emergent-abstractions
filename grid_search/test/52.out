# ["--attributes=5", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=60", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6721420288085938, "acc": 0.6138220429420471, "length": 3.5235166549682617, "mode": "train", "epoch": 1}
{"loss": 0.6569147706031799, "acc": 0.6320312023162842, "length": 4.0, "mode": "test", "epoch": 1}
{"loss": 0.6581835150718689, "acc": 0.6282744407653809, "length": 3.4577114582061768, "mode": "train", "epoch": 2}
{"loss": 0.6407345533370972, "acc": 0.631878674030304, "length": 3.96484375, "mode": "test", "epoch": 2}
{"loss": 0.6249167323112488, "acc": 0.6631306409835815, "length": 3.776845693588257, "mode": "train", "epoch": 3}
{"loss": 0.6060677170753479, "acc": 0.6814453601837158, "length": 4.0, "mode": "test", "epoch": 3}
{"loss": 0.5938352942466736, "acc": 0.6782512664794922, "length": 3.94763445854187, "mode": "train", "epoch": 4}
{"loss": 0.6171769499778748, "acc": 0.6722655892372131, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.5834287405014038, "acc": 0.6817426085472107, "length": 3.970161199569702, "mode": "train", "epoch": 5}
{"loss": 0.6319108605384827, "acc": 0.6490234732627869, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5758404731750488, "acc": 0.6880972981452942, "length": 3.973414897918701, "mode": "train", "epoch": 6}
{"loss": 0.6488794684410095, "acc": 0.6564452648162842, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.5709710121154785, "acc": 0.6945881247520447, "length": 3.980358362197876, "mode": "train", "epoch": 7}
{"loss": 0.6149765849113464, "acc": 0.6802734732627869, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.5633515119552612, "acc": 0.696212887763977, "length": 3.9900763034820557, "mode": "train", "epoch": 8}
{"loss": 0.5826225280761719, "acc": 0.6929688453674316, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.5503963232040405, "acc": 0.7055419683456421, "length": 3.9931368827819824, "mode": "train", "epoch": 9}
{"loss": 0.5831559896469116, "acc": 0.6986327767372131, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.5411801338195801, "acc": 0.7155841588973999, "length": 3.988996744155884, "mode": "train", "epoch": 10}
{"loss": 0.5926728248596191, "acc": 0.6843749284744263, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.5205358862876892, "acc": 0.7441886067390442, "length": 3.983813762664795, "mode": "train", "epoch": 11}
{"loss": 0.5604634881019592, "acc": 0.7310547232627869, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.49875807762145996, "acc": 0.7518406510353088, "length": 3.9877047538757324, "mode": "train", "epoch": 12}
{"loss": 0.5487983822822571, "acc": 0.7275390625, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.4861440360546112, "acc": 0.7594781517982483, "length": 3.995493173599243, "mode": "train", "epoch": 13}
{"loss": 0.5379407405853271, "acc": 0.7404297590255737, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.46525856852531433, "acc": 0.7743310928344727, "length": 3.9945170879364014, "mode": "train", "epoch": 14}
{"loss": 0.5332454442977905, "acc": 0.7511719465255737, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.45291104912757874, "acc": 0.7912901043891907, "length": 3.9964797496795654, "mode": "train", "epoch": 15}
{"loss": 0.5343413949012756, "acc": 0.7593750357627869, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.444302499294281, "acc": 0.797958254814148, "length": 3.9959805011749268, "mode": "train", "epoch": 16}
{"loss": 0.5080404281616211, "acc": 0.7693358659744263, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.42739778757095337, "acc": 0.8086323738098145, "length": 3.996321201324463, "mode": "train", "epoch": 17}
{"loss": 0.5175127983093262, "acc": 0.7650390267372131, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.42017862200737, "acc": 0.8101860880851746, "length": 3.997593641281128, "mode": "train", "epoch": 18}
{"loss": 0.507781982421875, "acc": 0.7703124284744263, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.4179235100746155, "acc": 0.8131780624389648, "length": 3.9976675510406494, "mode": "train", "epoch": 19}
{"loss": 0.5069997906684875, "acc": 0.7783203125, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.4018453061580658, "acc": 0.8221679925918579, "length": 3.997819662094116, "mode": "train", "epoch": 20}
{"loss": 0.4846647083759308, "acc": 0.7802734375, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3964853286743164, "acc": 0.8256545662879944, "length": 3.998297929763794, "mode": "train", "epoch": 21}
{"loss": 0.4822882413864136, "acc": 0.78515625, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.3934059739112854, "acc": 0.8299221992492676, "length": 3.999244213104248, "mode": "train", "epoch": 22}
{"loss": 0.4746658504009247, "acc": 0.7855468392372131, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.38861605525016785, "acc": 0.8310577273368835, "length": 3.9970197677612305, "mode": "train", "epoch": 23}
{"loss": 0.47720029950141907, "acc": 0.7939453125, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.38393473625183105, "acc": 0.8308694958686829, "length": 3.999127149581909, "mode": "train", "epoch": 24}
{"loss": 0.4528798758983612, "acc": 0.8050780892372131, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.37720009684562683, "acc": 0.8355937004089355, "length": 3.9996113777160645, "mode": "train", "epoch": 25}
{"loss": 0.4623262286186218, "acc": 0.8044921159744263, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3742806613445282, "acc": 0.8381809592247009, "length": 3.9993746280670166, "mode": "train", "epoch": 26}
{"loss": 0.4495818316936493, "acc": 0.8027344346046448, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3672358989715576, "acc": 0.8450917601585388, "length": 3.999527931213379, "mode": "train", "epoch": 27}
{"loss": 0.449939101934433, "acc": 0.8050780892372131, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3601193130016327, "acc": 0.8469269871711731, "length": 3.999752998352051, "mode": "train", "epoch": 28}
{"loss": 0.4395396411418915, "acc": 0.8173828721046448, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.3542605936527252, "acc": 0.8517566323280334, "length": 3.9989986419677734, "mode": "train", "epoch": 29}
{"loss": 0.449749231338501, "acc": 0.8109375238418579, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.3529626727104187, "acc": 0.8533007502555847, "length": 3.9996721744537354, "mode": "train", "epoch": 30}
{"loss": 0.43600213527679443, "acc": 0.8236327767372131, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3452584445476532, "acc": 0.8589677214622498, "length": 3.9997594356536865, "mode": "train", "epoch": 31}
{"loss": 0.43696340918540955, "acc": 0.8230468034744263, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.34186381101608276, "acc": 0.8613302707672119, "length": 3.999866485595703, "mode": "train", "epoch": 32}
{"loss": 0.452165812253952, "acc": 0.8078125715255737, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.3363989591598511, "acc": 0.8621572256088257, "length": 3.9995203018188477, "mode": "train", "epoch": 33}
{"loss": 0.42199477553367615, "acc": 0.8248047232627869, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.3385547995567322, "acc": 0.862040638923645, "length": 3.999907970428467, "mode": "train", "epoch": 34}
{"loss": 0.4307299554347992, "acc": 0.8248047232627869, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.33272895216941833, "acc": 0.8661133646965027, "length": 3.999880075454712, "mode": "train", "epoch": 35}
{"loss": 0.4293837547302246, "acc": 0.8185547590255737, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.32690852880477905, "acc": 0.8688892722129822, "length": 3.9999587535858154, "mode": "train", "epoch": 36}
{"loss": 0.42090609669685364, "acc": 0.8279295563697815, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.32470545172691345, "acc": 0.8702471852302551, "length": 3.9998741149902344, "mode": "train", "epoch": 37}
{"loss": 0.4175560474395752, "acc": 0.8296875357627869, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.3222469985485077, "acc": 0.871014416217804, "length": 3.9999682903289795, "mode": "train", "epoch": 38}
{"loss": 0.4259541928768158, "acc": 0.8228515982627869, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.3229372203350067, "acc": 0.8699523210525513, "length": 3.999957323074341, "mode": "train", "epoch": 39}
{"loss": 0.41936159133911133, "acc": 0.8294921517372131, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.31562572717666626, "acc": 0.8746128678321838, "length": 3.9998881816864014, "mode": "train", "epoch": 40}
{"loss": 0.42980754375457764, "acc": 0.8292969465255737, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.31036055088043213, "acc": 0.8750883340835571, "length": 3.999985933303833, "mode": "train", "epoch": 41}
{"loss": 0.4233211874961853, "acc": 0.8318358659744263, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.3111400902271271, "acc": 0.8749106526374817, "length": 3.999945878982544, "mode": "train", "epoch": 42}
{"loss": 0.40521904826164246, "acc": 0.8359375, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.32212722301483154, "acc": 0.8702471852302551, "length": 3.999948740005493, "mode": "train", "epoch": 43}
{"loss": 0.40530815720558167, "acc": 0.8347655534744263, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.30426451563835144, "acc": 0.8782759308815002, "length": 3.999976873397827, "mode": "train", "epoch": 44}
{"loss": 0.4138038754463196, "acc": 0.8271483778953552, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.30440521240234375, "acc": 0.8801057934761047, "length": 3.9999728202819824, "mode": "train", "epoch": 45}
{"loss": 0.41685280203819275, "acc": 0.8337889909744263, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.29479414224624634, "acc": 0.8846514225006104, "length": 3.9999892711639404, "mode": "train", "epoch": 46}
{"loss": 0.4092482328414917, "acc": 0.8287109732627869, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.299664705991745, "acc": 0.8788025975227356, "length": 3.999554395675659, "mode": "train", "epoch": 47}
{"loss": 0.4260677397251129, "acc": 0.8345703482627869, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.2937565743923187, "acc": 0.882172167301178, "length": 3.9999935626983643, "mode": "train", "epoch": 48}
{"loss": 0.40657174587249756, "acc": 0.8335937857627869, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.2889213263988495, "acc": 0.8860679864883423, "length": 3.99997615814209, "mode": "train", "epoch": 49}
{"loss": 0.4044222831726074, "acc": 0.8386718034744263, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.29734426736831665, "acc": 0.8821132779121399, "length": 3.9999990463256836, "mode": "train", "epoch": 50}
{"loss": 0.39888426661491394, "acc": 0.8365234732627869, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.28872430324554443, "acc": 0.8840612173080444, "length": 3.9999940395355225, "mode": "train", "epoch": 51}
{"loss": 0.39849933981895447, "acc": 0.8433593511581421, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.2883453667163849, "acc": 0.887008547782898, "length": 3.998786449432373, "mode": "train", "epoch": 52}
{"loss": 0.40395060181617737, "acc": 0.8361328840255737, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2953037917613983, "acc": 0.8835299611091614, "length": 3.999981641769409, "mode": "train", "epoch": 53}
{"loss": 0.40143314003944397, "acc": 0.837890625, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2879834473133087, "acc": 0.8870125412940979, "length": 3.999985933303833, "mode": "train", "epoch": 54}
{"loss": 0.4217928946018219, "acc": 0.8351563215255737, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.28092798590660095, "acc": 0.8874260187149048, "length": 3.999998092651367, "mode": "train", "epoch": 55}
{"loss": 0.3913511633872986, "acc": 0.8466796875, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.27241960167884827, "acc": 0.8947461247444153, "length": 3.9999983310699463, "mode": "train", "epoch": 56}
{"loss": 0.39721670746803284, "acc": 0.8367186784744263, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2757212221622467, "acc": 0.8919125199317932, "length": 3.999997138977051, "mode": "train", "epoch": 57}
{"loss": 0.39005425572395325, "acc": 0.8462891578674316, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.272088885307312, "acc": 0.8935047388076782, "length": 3.99995493888855, "mode": "train", "epoch": 58}
{"loss": 0.4093259871006012, "acc": 0.8427734375, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.27109596133232117, "acc": 0.8929750323295593, "length": 3.9999961853027344, "mode": "train", "epoch": 59}
{"loss": 0.39338451623916626, "acc": 0.8492187857627869, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.26899290084838867, "acc": 0.8945682644844055, "length": 3.999986410140991, "mode": "train", "epoch": 60}
{"loss": 0.39622578024864197, "acc": 0.8419922590255737, "length": 4.0, "mode": "test", "epoch": 60}
test accuracy: 0.7603225708007812
